src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -341,42 +341,64 @@ public HttpClientSettingsKey getHttpClientKey() throws SnowflakeSQLException {
       String httpsProxyHost = systemGetProperty("https.proxyHost");
       String httpsProxyPort = systemGetProperty("https.proxyPort");
       String noProxy = systemGetEnv("NO_PROXY");
+      String nonProxyHosts = systemGetProperty("http.nonProxyHosts");
```

Comment:
So this is the only http property that we missed?

Reply:
Yes! This is an extra thing I noticed while editing the code. The main change is adding a new HttpClientSettingsKey for http JVM proxies instead of only adding a new HttpClientSettingsKey for https JVM proxies.

Reply:
Have we documented this and communicated with the customers somewhere?

Reply:
This is a bug fix. We already document that we support both of these parameters, but didn't actually. So it will be in the release notes as a bug fix.